### PR TITLE
SLE 15 SP2 SQLAchemy version doesn't support hide_parameters

### PR DESCRIPTION
**Pull request by rtamalin**
Try create_engine() with hide_parameters, and if it failed with a
TypeError and hide_paramaters in the exception message then try
again without the hide_parameters argument.

### CODE REVIEW

File: pint_server/database.py [hunk 1]
```diff
@@ -188,8 +188,19 @@ def init_db(dbconfig=None, outputfile=None, echo=None,
...
-                           echo=echo, hide_parameters=hide_parameters)
+    # TODO(rtamalin): Remove this try/except hackery once we move forward
+    # to being based on SLE 15 SP3 or later.
+    try:
+        engine = create_engine(engine_url, convert_unicode=True,
+                               echo=echo, hide_parameters=hide_parameters)
+    except TypeError as e:
+        # If we failed because of the hide_parameters argument then
+        # try again without it.
+        if 'hide_paramaters' in e.msg:
```
Comment (mbelur) [inline pint_server/database.py:199]: typo in hide_parameters

Comment (rtamalin) [reply pint_server/database.py]: Should be fixed now; thanks for spotting!

**Review by guangyee — COMMENTED** [review]
looks good

**Review by mbelur — APPROVED** [review]
LGTM!

**Comment by guangyee** [comment]
don't merge yet. Getting this error while testing the code.

`AttributeError: 'TypeError' object has no attribute 'msg'`

File: pint_server/database.py [hunk 2]
```diff
@@ -188,8 +188,19 @@ def init_db(dbconfig=None, outputfile=None, echo=None,
...
-                           echo=echo, hide_parameters=hide_parameters)
+    # TODO(rtamalin): Remove this try/except hackery once we move forward
+    # to being based on SLE 15 SP3 or later.
+    try:
+        engine = create_engine(engine_url, convert_unicode=True,
+                               echo=echo, hide_parameters=hide_parameters)
+    except TypeError as e:
+        # If we failed because of the hide_parameters argument then
+        # try again without it.
+        if 'hide_parameters' in e.msg:
```
Comment (guangyee) [inline pint_server/database.py:199]: try this instead:

`if 'hide_parameters' in str(e):`

**Review by guangyee — CHANGES_REQUESTED** [review]
see my comment

**Comment by rtamalin** [comment]
> don't merge yet. Getting this error while testing the code.
> 
> `AttributeError: 'TypeError' object has no attribute 'msg'`

Looks like I need to check for the string in `str(e)`


Which you just posted ;-)

File: pint_server/database.py (hunk 2 above)
Comment (rtamalin) [reply pint_server/database.py]: Should be fixed now

**Review by guangyee — APPROVED** [review]
LGTM, tested in my local env

**Pull request merged**